package.json

```diff
@@ -15,5 +16,9 @@
   "bugs": {
     "url": "https://github.com/dashevo/govobject-proposal/issues"
   },
-  "homepage": "https://github.com/dashevo/govobject-proposal#readme"
+  "homepage": "https://github.com/dashevo/govobject-proposal#readme",
+  "devDependencies": {
+    "bitcore-lib-dash": "github:dashevo/bitcore-lib-dash",
```

Comment:
Yes that what I mentionned on the point Nota 2, but we first need to define a tag on the github of bitcore-lib-dash, or regain access from NPM